bin/collator/src/cli.rs

```diff
@@ -168,6 +176,7 @@ impl RelayChainCli {
 }
 
 /// EVM tracing CLI flags.
+#[cfg(feature = "evm-tracing")]
```

Comment:
Why not move all `evm-tracing` related type into a separate module?
The code would be much cleaner.

Reply:
I didn’t change anything here, just add few preprocessor conditions; all comments are fair for 5.0 without this PR.

Reply:
I commented what you changed - adding the feature check :man_shrugging:.